src/components/App.tsx

```diff
@@ -3580,9 +3580,18 @@ class App extends React.Component<ExcalidrawProps, AppState> {
 
     const file = event.dataTransfer?.files[0];
     if (
-      file?.type === "application/json" ||
-      file?.name.endsWith(".excalidraw")
+      file?.type === MIME_TYPES.excalidrawlib ||
+      file?.name?.endsWith(".excalidrawlib")
```

Comment:
We should probably move the allowed extensions into `constants.ts`.

Reply:
I'll do it in a separate PR coz I'm not sure what's the best way to do it.